onboarding_project/squad_pantry_app/models.py

```diff
@@ -71,12 +71,31 @@ def save(self, *args, **kwargs):
             self.closed_at = timezone.now()
         super(Order, self).save(*args, **kwargs)
 
+    def check_limit(self, limit, user_id):
```

Comment:
- This should be a class method, not an instance method. This logic has nothing to do with a specific instance of an order
- We don't need `user_id`
- Don't save an order at all. Prevent the order from getting inserted into the db

Reply:
Done!